astropy/tests/pytest_plugins.py

```diff
@@ -35,10 +35,9 @@
 from ..utils.argparse import writeable_directory
 from ..utils.introspection import resolve_name
 
-# Needed for Python 2.6 compatibility
 try:
```

Comment:
No - it's actually needed for Python 2.7 too, so I just removed the comment because it was wrong and added a comment on the except line :)